spec/spec_helper.rb

```diff
@@ -92,4 +92,8 @@
   #   # test failures related to randomization by passing the same `--seed` value
   #   # as the one that triggered the failure.
   #   Kernel.srand config.seed
+
+  config.before(:each, type: :system) do
```

Comment:
should this be in the spec_helper? Can you run a system spec without Rails loaded? :)